index.html

```diff
@@ -26,162 +26,189 @@
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="made-with">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </li>
-      <li class="papd">
-        <h3>Profesional Art<br>Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="made-with">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button" class="btn2">See Project</button>
-      </li>
-    </ul>
-  </section>
-
-  <!-- About section -->
-  <section class="about">
-    <section class="about-me">
-      <h2>About me</h2>
-      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      <button type="button" class="btn1">Get My Resume</button>
+  <main>
+    <section class="headline">
+      <div class="headline-header">
+        <h1 class="headline-title">Hey there.<br>I'm Onesis</h1>
+        <h2>I am a frontend Developer</h2>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <ul class="social">
+        <li class="social-icon"><a href="#"><i class="fab fa-github"></i></a></li>
+        <li class="social-icon"><a href="#"><i class="fab fa-linkedin"></i></a></li>
+        <li class="social-icon"><a href="#"><i class="fab fa-angellist"></i></a></li>
+        <li class="social-icon"><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li class="social-icon"><a href="#"><i class="fab fa-medium"></i></a></li>
+      </ul>
     </section>
-    <hr>
-    <section class="knowledge">
-      <div class="knowledge-titles">
-        <img src="img/Rectangle-55.png" alt="">
-        <h3>Language</h3>
+
+    <!-- Works section -->
+
+    <section class="my-works">
+      <div class="works-heading">
+        <h2 class="works-title">My Recent Works</h2>
+        <hr class="separate">
       </div>
-        <ul>
-          <li>Javascript</li>
-          <li>Ruby</li>
-          <li>Html</li>
-          <li>Css</li>
+      <div class="projects">
+        <div class="multipost">
+          <img src="img/multipost-Img.jpg" alt="multi-post project">
+          <div>
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] According to the figma design, the `hr` is supposed to take the rest of the page width.
- [ ] The paragraph, `A daily selection of .....` should not be bold. You're expected to follow the given design's font, color and typography

Yours | Original 
----- | -----
![image](https://user-images.githubusercontent.com/16058470/121672782-19bc0100-caa8-11eb-9ec3-a3a76d539264.png) | ![image](https://user-images.githubusercontent.com/16058470/121672865-35bfa280-caa8-11eb-9ff7-5d3f34bac4e9.png)

Reply:
Hi Elijah!

May you please be more specific with the hr required change because in the 1440px width view of my project the hr line looks exactly as it should be based on the Figma template which is also 1440px width.

The screenshot you shared is not the 1440px width view.

I'll share a screenshot of how mine looks for 1440px:

![micr](https://user-images.githubusercontent.com/64618543/121697495-1ff7b080-ca9b-11eb-9c20-445be71f0259.png)